question16/main.js

```diff
@@ -83,21 +82,57 @@ const createTitles = ({ articles }) => {
     const titleList = document.createElement('li');
     const articleLink = document.createElement('a');
     articleLink.href = "#";
-    articleLink.textContent = articleTitle; 
+    articleLink.textContent = articleTitle;
+
+    // Display new icon
+    const elapsedTime = getElapsedDays(articles[i]);
```

Comment:
```suggestion
    const elapsedTime = getElapsedDays(articles[i].date);
```

Reply:
Thank you for pointing it out! I fixed it.

Latest commit : https://github.com/csakyo/homework/pull/30/commits/ccad1949a6adfc7ecda8ae44fcab7c62dce7d1a7